mlrun/common/schemas/alert.py

```diff
@@ -200,3 +200,30 @@ def templates_differ(self, other):
             or self.reset_policy != other.reset_policy
             or self.criteria != other.criteria
         )
+
+
+class AlertActivation(pydantic.BaseModel):
+    name: str
+    project: str
+    severity: AlertSeverity
+    activation_time: datetime
+    entity_id: str
+    entity_kind: EventEntityKind
+    criteria: AlertCriteria
+    event_kind: EventKind
+    number_of_events: int
+    notifications: list[NotificationState]
+
+
+class PartitionInterval(StrEnum):
+    DAY = "DAY"
+    MONTH = "MONTH"
+    YEARWEEK = "YEARWEEK"
+
+    @classmethod
+    def is_valid(cls, value: str) -> bool:
+        return value in cls._value2member_map_
+
+    @classmethod
+    def valid_intervals(cls) -> list:
+        return list(cls._value2member_map_.keys())
```

Comment:
Duplicate of https://github.com/mlrun/mlrun/blob/20a496cee30ef173dde1518987b8339b15d39ed2/mlrun/common/schemas/partition.py#L20